askar-crypto/Cargo.toml

```diff
@@ -48,32 +50,62 @@ harness = false
 [dependencies]
 aead = "0.5"
 aes-core = { package = "aes", version = "0.8", default-features = false, optional = true }
-aes-gcm = { version = "0.10", default-features = false, features = ["aes"], optional = true }
+aes-gcm = { version = "0.10", default-features = false, features = [
+    "aes",
+], optional = true }
 arbitrary = { version = "1.0", optional = true, features = ["derive"] }
-argon2 = { version = "0.5", default-features = false, features = ["alloc", "password-hash"], optional = true }
+argon2 = { version = "0.5", default-features = false, features = [
+    "alloc",
+    "password-hash",
+], optional = true }
 base64 = { version = "0.21", default-features = false }
 blake2 = { version = "0.10", default-features = false }
 block-modes = { version = "0.9", default-features = false, optional = true }
-bls12_381 = { version = "0.8", default-features = false, features = ["groups", "zeroize"], optional = true }
+bls12_381 = { version = "0.8", default-features = false, features = [
+    "groups",
+    "zeroize",
+], optional = true }
 cbc = { version = "0.1", default-features = false, optional = true }
-chacha20 = { version = "0.9" }  # should match dependency of chacha20poly1305
+chacha20 = { version = "0.9" } # should match dependency of chacha20poly1305
```

Comment:
not a change you introduced, but `chacha20` does not match `chacha20poly1305`. Maybe we should remove the comment? The latest `chacha20` is 0.9.1 and `chacha20poly1305` is on `0.10.1`.

Reply:
This is correct, although the comment could maybe be clearer. Because we use both both of these crates (and there isn't a re-export available), it's important to make sure that the version of `chacha20` matches the version depended upon by `chacha20poly1305`.